MANIPULATION_ENABLE: true
PICK_GROUPS: [arm] #, whole_body, whole_body_rotational
ARM_GROUP: "arm"
ARM_JOINTS: [joint1, joint2, joint3, joint4, joint5, joint6]
ROBOT_MAX_RANGE_XY: 0.525
# ARM_GRASP: [-1.57, 0.0, 0.0, -1.57, -1.57]
ARM_GRASP: [-1.57, -0.785, -0.785, 0, 1.57, -2.3562]
ARM_INIT: [-1.57, -1.39626, -0.174533, 0.0, 1.57, -0.7854]
ARM_HOME: [0.0, 0.0, 0.0, -1.5708, 1.5708, 0.7854]
ARM_PREGRASP: [-1.57, 0, -3.14, 0, 1.8326, 0.7854]
ARM_CARTESIAN_PREGRASP_VERTICAL: [-1.5707963705062866, -0.6108652353286743, -2.181661605834961, 0.0, 2.7925267219543457, -2.356194496154785]
ARM_CARTESIAN_PREGRASP_HORIZONTAL: [-1.570796251296997, -0.4812801480293274, -2.0814273357391357, -8.003445373105933e-08, 0.9919110536575317, 0.7853982448577881]
ARM_CARTESIAN_POSTGRASP_VERTICAL: [-1.5708004236221313, -0.31413206458091736, -2.22076416015625, 5.999552740831859e-06, 2.5305960178375244, -2.3507955074310303]
#ARM_PREGRASP: [-1.57, -0.6108, -0.4363, 0, -0.1745, 0.7854]
HEAD_ENABLE: false
HEAD_GROUP: "head"
HEAD_JOINTS: [Cam1Rev1, Cam1Rev2]
HEAD_LOOK_DOWN: [0.0, 0.0]
HEAD_LOOK_FRONT: [0.0, 0.0]
HEAD_LEFT: [0.0, 0.0]
HEAD_RIGHT: [0.0, 0.0]
GRIPPER_GROUP: "gripper"
GRIPPER_JOINTS: [ "Rev1Servo", "Rev2Servo"]
GRIPPER_OPEN: [-0.5, 0.5]
GRIPPER_CLOSED: [0.0, 0.0]
GRASP_POSTURES_FRAME_ID: "end_effector_link"
ALLOW_CONTACT: [
  "Servo1",
  "Servo2",
  "Dedo1",
  "Dedo2"
]
CAMERA_DEPTH_FRAME: zed2_left_camera_optical_frame
CAMERA_PC_FRAME: zed2_left_camera_frame
